frontend/cs/scs/api.go

```diff
@@ -559,6 +561,34 @@ func (builder *builder) Compiler() frontend.Compiler {
 	return builder
 }
 
+func scsBsb22CommitmentHintPlaceholder(*big.Int, []*big.Int, []*big.Int) error {
+	return errors.New("placeholder - should never be called")
+}
+
+func (builder *builder) Commit(v ...frontend.Variable) (frontend.Variable, error) {
+
+	committed := make([]int, len(v))
+
+	for i, vI := range v { // TODO @Tabaie Perf; If public, just hash it
+		vINeg := builder.Neg(vI).(expr.Term)
```

Comment:
Perhaps we should handle the error here if the variable is constant (or anything else than a Term) ? Otherwise we get a message like this: `parse circuit: interface conversion: frontend.Variable is *big.Int, not expr.Term`, not sure if it's very explicit.

Reply:
Addressed in new commit 👍